test-case/check-alsabat.sh

```diff
@@ -73,6 +73,8 @@ frequency=${OPT_VAL['F']}
 sigmak=${OPT_VAL['k']}
 frames=${OPT_VAL['n']}
 
+start_test
+
```

Comment:
In commit message, what is the meaning of the opening sentence in relation to moving the start_test from lib.sh to test cases?
`Stop unconditionally imposing start_test() to everything sourcing lib.sh`